tests/engagement/conftest.py

```diff
@@ -1,5 +1,28 @@
+import functools
+
 import pytest
-from mozetl.churn import release
+
+from data import main_summary_sample, main_summary_schema, new_profile_sample, \
+    new_profile_schema
```

Comment:
No backslashes in imports please. It's strongly suggested in PEP8 to use Python's line continuation in parentheses, and also use absolute imports:
```
from .data import (
    main_summary_sample, main_summary_schema, new_profile_sample
    new_profile_schema
)
```

Or alternatively import data and use it as a namespace in the code:

```python
from . import data
# ..
print(data.new_profile_schema)
```